ext/net/ops_tls.rs

```diff
@@ -190,6 +190,13 @@ impl TlsStream {
   fn poll_handshake(&mut self, cx: &mut Context<'_>) -> Poll<io::Result<()>> {
     self.inner_mut().poll_handshake(cx)
   }
+
+  fn get_alpn_protocol(&mut self) -> Option<String> {
+    match self.inner_mut().tls.get_alpn_protocol() {
+      None => None,
+      Some(s) => Some(std::str::from_utf8(s).unwrap().to_string())
```

Comment:
The unwrap here is not good. It is a vector for a denial of service. The string should be returned as a `ByteString` (this is a string where each char is some value between 0-255) - not a utf8 string.

Reply:
Right. I've updated the PR to use ByteString.